api/api_test.go

```diff
@@ -2762,3 +2762,37 @@ func TestServer_GetActPoolActions(t *testing.T) {
 	_, err = svr.GetActPoolActions(context.Background(), &iotexapi.GetActPoolActionsRequest{ActionHashes: []string{hex.EncodeToString(h3[:])}})
 	require.Error(err)
 }
+
+func TestServer_GetEstimateGasSpecial(t *testing.T) {
+	require := require.New(t)
+	cfg := newConfig(t)
+
+	svr, bfIndexFile, err := createServer(cfg, true)
+	require.NoError(err)
+	defer func() {
+		testutil.CleanupPath(t, bfIndexFile)
+	}()
+
+	// deploy self-desturct contract
```

Comment:
i may miss the context, what's special about this test?

couple of days ago, there's already a check-in about self-destruct https://github.com/iotexproject/iotex-core/commit/dcf27b4902cebda891b1edbf5d817515f933bb55

Reply:
discussed with Haixiang, fix should go into `api.go`